src/adr/articles/0009-integration-ualpa.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: layouts/adr.njk
+title: "Integration with Use a LPA"
+weight: 1
+date: 2022-03-16
+review_in: 12 months
+tags:  
+    - adr
+    - open_standards
+    - apis
+    - language
+    - integration
+areas_of_coverage: ["Digital Service"]
+status: "accepted"
+contributors: ["John Nolan"]
+adr_number: "0009"
+---
+
+## Context
+
+Our [Use a LPA service](https://use-lasting-power-of-attorney.service.gov.uk/home) is the latest service built by the OPG. We want to build any new features around this service moving forward, taking advantage of the work done on it.
```

Comment:
Maybe the ADR is better framed as Modernised LPA must provide functionality (API) to allow Use An LPA to still function with new style LPAs. Is the decision here that we want UALPA to still be the primary interface for sharing/viewing in the new world? Or is it that we want to push the current business logic that's in UALPA further back so it can be in a shared service with modernise?

Are we conflating public facing window on data with the data itself int his ADR?

Lots of questions. Sorry.

Reply:
Lots of good questions and thank you for asking them, this is one of the big one and most unknown as of yet. Will have a think in a bit get push an update that addresses the above

Reply:
Have changed the context and a few other parts to be more specific. After recent discussions around domains and responsibilities, I think the new context covers the comments you left. Feels more purposeful now, but let me know if its still not quite hitting the mark